src/plopp/graphics/outline.py

```diff
@@ -38,8 +53,26 @@ def _make_sprite(string, position, color="black", size=1.0):
 
 
 class Outline(p3.Group):
+    """
+    Create an object that draws a rectangular outline, given some limits for the XYZ
+    dimensions. Along the lower edges of the cube are added some tick labels and axes
+    labels, according to the dimension extents, dimension names, and units given in
+    the limits.
+
+    Parameters
+    ----------
+    limits:
+        A tuple of variables, each of length 2, which contain the lower and upper bounds
+        for the outline. Each variable also has a dimension, which is to be used as the
+        dimension for that direction, as well as a unit, which will be used to label
+        the corresponding axis.
+    tick_size:
+        A number to scale the tick size.
+    """
 
-    def __init__(self, limits, tick_size=None):
+    def __init__(self,
+                 limits: Tuple[Variable, Variable, Variable],
+                 tick_size: float = None):
 
         center = [var.mean().value for var in limits]
```

Comment:
Using this below causes type error `Expected type 'Tuple[float, float, float]', got 'list' instead `